### ISSUE TITLE
limel-form: Empty string is different depending on required true/false

### ISSUE THREAD

**Issue by TommyLindh2** [Lundalogik/lime-elements#1598]
## Current behavior

If you have limel-form with a property with type `string` it returns different "empty value" depending on if the property is set to required or not.

Required => `null`
No Required => `""`

Here's an example made in the documentation

![image](https://user-images.githubusercontent.com/1814702/153581868-42f9d97c-5540-4042-923d-34d89b4207be.png)

1. See above screen shot.

## Expected behavior

It would be nice if the form always returned an empty string (`""`) when it's empty

## Environment

- **lime-elements** version: 34.1.0

**Comment by TommyLindh2** [comment]
@BjornOstberg if you want to follow

**Comment by adrianschmidt** [comment]
Yeah, this seems like an annoying inconsistency, but I wonder if it's perhaps necessary to make form validation work? If not, I think we should fix it…

@jgroth: Do you think this might be form validation related?

**Comment by devbymadde** [comment]
Closed due to age. Please reopen if the issue still persists.
